.github/workflows/tests.yaml

```diff
@@ -10,36 +10,17 @@ on:
 
 jobs:
   tests:
-
     runs-on: ubuntu-latest
     timeout-minutes: 10
 
     steps:
-    - name: Clone oresat-c3 repository
-      uses: actions/checkout@v2
-
-    - name: Clone oresat-configs repository
```

Comment:
It is better to build the latest version of `oresat-configs`, as the last release may not have changes needed by this project. Main case is where new C3 object are added to the OD that the code now uses.

Reply:
Would it make sense to do small `oresat-configs` point releases at a quicker cadence in that case? My concern is that while you and I both can easily and know to install a dev version of `oresat-configs`, someone starting on the project is going to clone the repo and `pip install` the dependencies and then get stuck because it doesn't work.

Reply:
I can see where that could be confusing, but we can point to the script in the `oresat-configs` repo for the fix.

I don't think having a quicker cadence wouldn't help, as adding new code that depends on a new object would then require a new release just for the workflow to pass. Making a new release each time an object is added little much. Waiting for the next release isn't great either.